cmd/generic-handlers.go

```diff
@@ -65,10 +65,10 @@ func (h requestSizeLimitHandler) ServeHTTP(w http.ResponseWriter, r *http.Reques
 	h.handler.ServeHTTP(w, r)
 }
 
-// Reserved bucket.
+// Reserved keyword and paths.
 const (
-	minioReservedBucket     = "minio"
-	minioReservedBucketPath = "/" + minioReservedBucket
+	minioReservedKeyword    = "minio"
```

Comment:
I felt it confusing to have two variables named similar to `reservedBuckets`. So changed `minioReservedBucket` to  `minioReservedKeyword`. Also `minioReservedBucket` is being used more as a reserved key word here. Hence the change here. 

I will add a global list instead of the local list in `isMinioReservedBucket`